tsdoc/src/parser/TSDocMessageId.ts

```diff
@@ -326,3 +329,67 @@ export const enum TSDocMessageId {
    */
   CodeSpanMissingDelimiter = 'tsdoc-code-span-missing-delimiter'
 }
+
+// Exposed via ParserMessage.isValidMessageId()
+export const allTsdocMessageIds: Set<string> = new Set<string>([
```

Comment:
You should be able to initialize this programaticially instead of copying+pasting all of these strings.

Reply:
I don't think it's possible, since it's a const enum.